lib/chef/knife/cookbook_site_install.rb

```diff
@@ -141,8 +141,17 @@ def download_cookbook_to(download_path)
 
       def extract_cookbook(upstream_file, version)
         ui.info("Uncompressing #{@cookbook_name} version #{version}.")
-        # FIXME: Detect if we have the bad tar from git on Windows: https://github.com/opscode/chef/issues/1753
         extract_command = "tar zxvf \"#{convert_path upstream_file}\""
+        if Chef::Platform.windows?
+          # FIXED: Detect if we have the bad tar from git on Windows: https://github.com/opscode/chef/issues/1753
+          tar_version = `tar --version`.tr("\n"," ")
```

Comment:
We don't use backticks for command execution in Chef given it has many many problems. See also, the use of `shell_out` right below this.